File: w1/home/multipleN/multipleN.c
#include <stdio.h>

int main() {
    int max;
    int divisor;
    
    scanf("%d %d", &max, &divisor);
    if ( divisor < 0 ) {
        divisor = -divisor;
    }
    
    max -= max % divisor;
    
    for ( int i = 0; i < max; i += divisor ) {
        printf("%d ", i);
    }
    printf("%d\n", max);
    
    return 0;
}

File: w4/php_dz/multipleN.php
<?php
    $max = rtrim(fgets(STDIN));
    $divisor = abs(rtrim(fgets(STDIN)));
    $out = "";

    $max -= $max % $divisor;

    for ( $i = 0; $i < $max; $i += $divisor ) {
        $out .= $i . " ";
    }

    echo $out . $max . PHP_EOL;
?>